libs/openchallenges/challenge-search/src/lib/challenge-search.component.ts

```diff
@@ -133,6 +135,7 @@ export class ChallengeSearchComponent
   // set default values
   defaultPageNumber = 0;
   defaultPageSize = 24;
+  defaultDropdownOptionSize = 5;
```

Comment:
```suggestion
  defaultDropdownOptionSize = 10;
```

Reply:
> I also noticed that this value control the number of elements/options displayed in the UI component at any time.

Does that come from our own design or from the design of the component from the library? Can we consider decoupling the two behavior? For example, displaying 10 elements but preloading 50 could make the scrolling smoother (less "bumps").

Reply:
`defaultDropdownOptionSize` (in challenge search component)) is the page size used to query how many items for dropdown items. It also feeds the input `optionsPerPage` for the dropdown component.

[virtualScrollItemSize](https://primeng.org/multiselect#api.multiselect.props.virtualScrollItemSize) (in dropdown component) is from default setting from the component, which is the height of item displayed. That's the only value you could control how many items displayed, but not great in my experience because we would need to adjust the item height to fit N items in a fixed-height scroller viewpoint.

`optionsPerPage` (in dropdown component) is I add to manually set how many items to load per page, as this component does not have the page size option. The `optionsPerPage` input will be used to calculate the page number for lazy loading, as well as the height of scroller viewpoint. 

With the dynamical change in scroller height now, your suggestion (preload 50 items, but display 10 items in scrollbar) is easy to achieve. I'll make the adjustment for review.